tests/test_send_mail.py

```diff
@@ -1,17 +1,30 @@
+# -*- coding: utf-8 -*-
 import pytest
 from send_email import send_email
 from config import DevConfig
+from faker import Faker
+import random
+import string
 
 
-@pytest.mark.skip(reason="This needs better test enviroment config.")
+fake = Faker()
+
+
+def text_generator(chars=string.ascii_letters + 'ąćęłóżź \n\t'):
+    size = random.randint(25, 40)
+    return ''.join(random.choice(chars) for _ in range(size))
+
+
+@pytest.mark.skip(reason="This needs better test environment config.")
```

Comment:
Is this test should be skipped? I commented decorator and it works

Reply:
I'm getting error, at this moment I'll leave this decorator and check out what's wrong with my computer later.